src/utils/VerifiableCredentialService.ts

```diff
@@ -0,0 +1,137 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import { BankAccountInfo, VerifiedCredential, VerifiedCredentialEvidence } from "../models/IVeriCredential";
+import { KmsJwtAdapter } from "./KmsJwtAdapter";
+import { ISessionItem } from "../models/ISessionItem";
+import { PersonIdentityNamePart } from "../models/PersonIdentityItem";
+import { AppError } from "./AppError";
+import { HttpCodesEnum } from "../models/enums/HttpCodesEnum";
+import { Constants } from "./Constants";
+import { randomUUID } from "crypto";
+import { CopCheckResult } from "../models/enums/CopCheckResult";
+import { MessageCodes } from "../models/enums/MessageCodes";
+
+export class VerifiableCredentialService {
```

Comment:
Should this live in the `services` folder?

Reply:
Moved to services & update test path